java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2245,6 +2247,33 @@ public T lastsave() {
         return getThis();
     }
 
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * To explicitly specify the flushing mode, use {@link #flushall(FlushMode)}.
```

Comment:
Do we want to include this line for the docs in ServerManagementCommands/ServerManagementClusterCommands as well or no?